contrib/devtools/security-check.py

```diff
@@ -8,15 +8,16 @@
 Otherwise the exit status will be 1 and it will log which executables failed which checks.
 Needs `objdump` (for PE) and `otool` (for MACHO).
 '''
+import shutil
 import subprocess
 import sys
 import os
 from typing import List, Optional
 
 import pixie
 
-OBJDUMP_CMD = os.getenv('OBJDUMP', '/usr/bin/objdump')
-OTOOL_CMD = os.getenv('OTOOL', '/usr/bin/otool')
+OBJDUMP_CMD = os.getenv('OBJDUMP', shutil.which('objdump'))
+OTOOL_CMD = os.getenv('OTOOL', shutil.which('otool'))
```

Comment:
If you are going to change these, you'll have to fixup the mypy issues (causing the lint job to fail):
```bash
contrib/devtools/symbol-check.py:198: error: List item 0 has incompatible type "Optional[str]"; expected "Union[bytes, str, _PathLike[Any]]"
contrib/devtools/symbol-check.py:219: error: List item 0 has incompatible type "Optional[str]"; expected "Union[bytes, str, _PathLike[Any]]"
Found 2 errors in 1 file (checked 189 source files)
^---- failure generated from test/lint/lint-python.sh
```

It's  unhappy because we've got an `Optional[str]` (from `shutil.which`) being added into the first argument of `subprocess.Popen`.

Reply:
Not sure how best to fix...

Naive fix:
```
diff --git a/contrib/devtools/symbol-check.py b/contrib/devtools/symbol-check.py
index 52f04e8cdf..e85f5b5fd4 100755
--- a/contrib/devtools/symbol-check.py
+++ b/contrib/devtools/symbol-check.py
@@ -52,9 +52,9 @@ IGNORE_EXPORTS = {
 '_edata', '_end', '__end__', '_init', '__bss_start', '__bss_start__', '_bss_end__', '__bss_end__', '_fini', '_IO_stdin_used', 'stdin', 'stdout', 'stderr',
 'environ', '_environ', '__environ',
 }
-CPPFILT_CMD = os.getenv('CPPFILT', shutil.which('c++filt'))
-OBJDUMP_CMD = os.getenv('OBJDUMP', shutil.which('objdump'))
-OTOOL_CMD = os.getenv('OTOOL', shutil.which('otool'))
+CPPFILT_CMD = os.getenv('CPPFILT', shutil.which('c++filt')) # type: ignore[list-item]
+OBJDUMP_CMD = os.getenv('OBJDUMP', shutil.which('objdump')) # type: ignore[list-item]
+OTOOL_CMD = os.getenv('OTOOL', shutil.which('otool')) # type: ignore[list-item]

 # Allowed NEEDED libraries
 ELF_ALLOWED_LIBRARIES = {
```